README.md

```diff
@@ -1,18 +1,24 @@
 **go-jsonschema is a tool to generate Go data types from [JSON Schema](http://json-schema.org/) definitions.**
 
-This tool generates Go data types and structs that corresponds to definitions in the schema, along with unmarshaling code that validates the input JSON according to the schema's validation rules.
+This tool generates Go data types and structs that corresponds to definitions in the schema, along with unmarshalling code that validates the input JSON according to the schema's validation rules.
 
 ## Installing
 
-* **Binary install**: Get a release [here](https://github.com/atombender/go-jsonschema/releases).
+* **Download**: Get a release [here](https://github.com/atombender/go-jsonschema/releases).
 
-* **From source**: Go 1.11 or later, with Go modules enabled, is advisable in order to get the right dependencies. To install:
+* **Install from source**: To install with Go 1.16+:
 
 ```shell
 $ go get github.com/atombender/go-jsonschema/...
 $ go install github.com/atombender/go-jsonschema/cmd/gojsonschema
 ```
 
+* **Install with Bingo**: To install with [Bingo](https://github.com/bwplotka/bingo):
+
+```shell
+$ bingo get github.com/atombender/go-jsonschema/cmd/gojsonschema
```

Comment:
I added this as personally that's how I'm using it, and it is handy to keep utilities out of go.mod